### ISSUE TITLE
Support for using emojis as tags 

### ISSUE THREAD

**Issue by xavierroy** [rothsandro/eleventy-notes#38]
I use a lot of emojis as tags. I use the map emoji (🗺️) for MoCs, books (📚) for books. 

Currently using emojis as tags breaks. Could you provide support for that?

**Comment by rothsandro** [comment]
Thanks for bringing this up. Tags can contain emojis when combined with text but it breaks when using only an emoji (or other special char) as tag. The reason is that each tag gets its own page and because the emoji is removed when generating the url, you end up with invalid/duplicate urls.

I see three possible solutions for this:

1. **Keep emojis**: We could keep the emoji in the url. Modern browsers support emojis in url but it could still break other software like read-it later apps if they don't support it. So this is probably not a good idea.
2. **Replace emojis**: We could replace emojis with their name, e.g. "📚" to "books" and "🗺️" to "world-map". But I couldn't find an up to date list of emojis with their name that would allow this transformation and new emojis are added regularly which makes it even harder. In addition, the name may not always be suitable, e.g. "📘" would result in "blue-book" but simply "book" would probably be better.
3. **Mapping tags**: We could allow renaming tags via `app.json`. In the example below, the notes can use the tag "🗺️" which would then be displayed as "🗺️ Map of Content" and result in the url `/tags/map-of-content`.

```json
{
  "tags": {
    "map": {
      "🗺️": "🗺️ Map of Content",
      "📚": "📚 Books"
    }
  }
}
```

What do you think?

**Comment by rothsandro** [comment]
Thanks for the feedback. I'll add support for this with the next minor version.

**Comment by rothsandro** [comment]
**Feature shipped** 🎉 It's available in v0.18.0 that has just been released.
